ebms-send-in/src/main/kotlin/no/nav/emottak/utbetaling/UtbetalingXmlMarshaller.kt

```diff
@@ -22,6 +28,12 @@ class XmlMarshaller {
             FinnBrukersUtbetalteYtelserRequest::class.java,
             FinnUtbetalingListe::class.java,
             FinnUtbetalingListeResponse::class.java,
+            FrikortsporringMengdeRequest::class.java,
```

Comment:
Kanskje legge disse et annet sted, for eksempel i FellesFormatXmlMarshaller eller en egen for frikort. De har ikke noe med utbetaling å gjøre.